services/openshiftbuilddeploymonitor/src/index.js

```diff
@@ -44,10 +46,13 @@ const messageConsumer = async msg => {
   } = JSON.parse(msg.content.toString())
 
   logger.verbose(`Received BuildDeployOpenshift monitoring task for project: ${projectName}, buildName: ${buildName}, openshiftProject: ${openshiftProject}, branch: ${branchName}, sha: ${sha}`);
-
-  const result = await getOpenShiftInfoForProject(projectName);
+  let result
+  result = await getOpenShiftInfoForProject(projectName);
   const projectOpenshift = result.project
 
+  result = await getEnvironmentByName(`${branchName}`, projectOpenshift.id)
```

Comment:
Let's use another variable than `result` as the variable is already used just above, a little bit saver. Also then we can actually use `const` for both variables. Again safer :)